main.css

```diff
@@ -1,26 +1,29 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;400;700&display=swap');
 
```

Comment:
Not A good Practice to include import in CSS file because code below the import code will execute after all the imports get load.